reddit_decider/__init__.py

```diff
@@ -411,7 +418,9 @@ def get_variant_for_identifier(
         self,
         experiment_name: str,
         identifier: str,
-        identifier_type: Literal["user_id", "device_id", "canonical_url"],
+        identifier_type: Literal[
+            "user_id", "device_id", "canonical_url", "subreddit_id", "ad_account_id"
```

Comment:
Does `identifier_type` need to include `business_id` too?

Reply:
Yes, yes it does. Thank you!